### DISCUSSION TITLE
Help using ffsubsync

### DISCUSSION THREAD

**Discussion by formlessdao** [smacke/subsync#146] [Q&A] [answered]
Hi, can you tell me please how can I make your app work for my case? I have transcribed a video in Word and now basically I have a .doc file with the full transcript. How can I use it with your app to generate a SRT formatted file with the correct timings?

**Comment by smacke** [comment, marked as answer]
You'll need to convert your .doc to srt format, for which I'm afraid ffsubsync isn't going to be helpful.
